README.md

```diff
@@ -297,6 +297,47 @@ LOKI_ROLE_ARN="${PRINCIPAL_ARN%:role/*}:role/$LOKI_ROLE"
 kubectl annotate ns loki iam.amazonaws.com/permitted="$LOKI_ROLE_ARN"
 ```
 
+#### Notes on specific configuration for clusters using Giant Swarm release >= v19.0.0 (i.e moving from KIAM to IRSA)
+
+From release v19.0.0, Giant Swarm clusters will use IRSA (Iam Roles for Service Accounts) to allow pods to access S3 buckets' resources. For more details concerning IRSA, you can refer to the [official documentation](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
```

Comment:
We could also have a link to our own doc for IRSA: https://docs.giantswarm.io/advanced/iam-roles-for-service-accounts